ops-bedrock/docker-compose-testnet.yml

```diff
@@ -122,8 +122,8 @@ services:
       OP_PROPOSER_L2OO_ADDRESS: "${L2OO_ADDRESS}"
       OP_PROPOSER_PPROF_ENABLED: "true"
       OP_PROPOSER_METRICS_ENABLED: "true"
-      OP_PROPOSER_ALLOW_NON_FINALIZED: "true"
-      OP_PROPOSER_NAMESPACE_ID: "e8e5f679bf7116cb"
+      OP_PROPOSER_ALLOW_NON_FINALIZED: "false"
```

Comment:
This seems unrelated.

Reply:
this was to test out the same changes as #133

Reply:
and would revert before merging

Reply:
https://github.com/celestiaorg/optimism/pull/132/commits/feaa7a9cfd22f8a5726d21fa633f3930524ac90c